app/cafe.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+
+from app.errors import (
+    NotWearingMaskError,
+    NotVaccinatedError,
+    OutdatedVaccineError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str | None:
+        if vaccine := visitor.get("vaccine"):
+            if datetime.date.today() > vaccine.get("expiration_date"):
+                raise OutdatedVaccineError
+        else:
+            raise NotVaccinatedError
```

Comment:
Same here, you should provide a descriptive error message when raising `NotVaccinatedError`. For example, `raise NotVaccinatedError('You are not vaccinated.')`.